cdbot/cogs/fun.py

```diff
@@ -32,7 +32,7 @@
     "dabato": "\N{THINKING FACE}",
     "kali": "\N{ONCOMING POLICE CAR}",
     "duck": "\N{DUCK}",
-    "revive": "nou",
+    "revive": "no u",
```

Comment:
Why is this reaction formatted differently to the ones above it?

Reply:
I added the space by mistake, will remove. It is because the current reaction is `NOU` but this would make it `NO U` with a spacing emoji.

Reply:
I get that, but I was wondering why it isn't `\N{NOU}` like the ones above it.

Reply:
Ah, because the `\N` is an emoji escape. For example, `\N{DUCK}` is the literal :duck: emoji.

Reply:
no u is a text response so it gets converted into emojis at a later stage.

Reply:
👍 Makes sense

Reply:
:eyes: